kyo-core/shared/src/main/scala/kyo/scheduler/Finalizers.scala

```diff
@@ -31,7 +31,7 @@ private[kyo] object Ensures:
                     arr.add(f)
                     arr
 
-        def remove(f: () => Unit): Ensures =
+        def remove(f: () => Unit): Finalizers =
```

Comment:
Not sure how often this API will be used directly, but worth a comment that the function reference must be the same (not just the value) IIUC.